docker-compose-dev.yml

```
@@ -0,0 +1,114 @@
+      - 6379:6379
+    networks:
+      - temporal-network
+
+volumes:
+  data1:
+
+
+networks:
+  temporal-network:
+    driver: bridge
+    name: temporal-network
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)